web/templates/note/index.html.eex

```diff
@@ -0,0 +1,10 @@
+<h1>Listing Notes</h1>
+
+<table class="table">
```

Comment:
I'll expand on that a little. Semantically, it doesn't really make sense for this to be a table. Also, while tables are nice because the default styles for a table will lay it out for you in a grid, if you want to extend/change the layout of this it can get crufty pretty fast. Tables were used a lot for styling in the 90s, before CSS was big, but shouldn't be used for that now that we have better tools for styling.